client-v2/package.json

```diff
@@ -121,7 +121,7 @@
     "redux-batched-actions": "^0.4.1",
     "redux-form": "^7.4.2",
     "redux-thunk": "^2.2.0",
-    "reselect": "^3.0.1",
+    "reselect": "^4.0.0",
```

Comment:
The main difference for 4.0 -- the updated typings, which lets selectors share props objects which have nothing in common. For the selector in this PR, we rely on { collectionId: string, collectionItemSet: string } in one selector and { articleFragmentId: string } in the next. Previously, when we used these selectors in createSelector the typings would complain that the signatures had nothing in common (true, but still valid) -- now, all is 👌